appspec.yml

```
@@ -14,3 +14,21 @@
       - location: scripts/start_container.sh
         timeout: 300
         runas: root
+
+#ps_Dev Branch
+  version: 0.0.0.0.0.0.1
```

YAMLlint (1.37.1)

[error] duplication of key "version" in mapping

(key-duplicates)